test_runner/batch_others/test_branching.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+import threading
+import pytest
+from fixtures.neon_fixtures import NeonEnv, PgBin, Postgres
+import time
+import random
+from fixtures.log_helper import log
+from performance.test_perf_pgbench import get_scales_matrix
+
+
+@pytest.mark.parametrize("n_branches", [10])
+@pytest.mark.parametrize("scale", get_scales_matrix(1))
+@pytest.mark.parametrize("ty", ["cascade", "flat"])
+def test_branching_with_pgbench(neon_simple_env: NeonEnv,
+                                pg_bin: PgBin,
+                                n_branches: int,
+                                scale: int,
+                                ty: str):
+    env = neon_simple_env
+
+    tenant, _ = env.neon_cli.create_tenant(
```

Comment:
```suggestion
    # Use aggressive GC and checkpoint settings, so that we also exercise GC during the test  
    tenant, _ = env.neon_cli.create_tenant(
```